.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   default:
     docker:
-    - image: circleci/golang:1.14.12
+    - image: circleci/golang:1.15.5
```

Comment:
[note] You can find the list of images here → https://circleci.com/docs/2.0/circleci-images/#go-golang